optimum/exporters/onnx/model_configs.py

```diff
@@ -650,6 +650,25 @@ class DonutSwinOnnxConfig(ViTOnnxConfig):
     pass
 
 
+class TimmResNextOnnxConfig(ViTOnnxConfig):
+    ATOL_FOR_VALIDATION = 1e-3
+
+    def rename_ambiguous_inputs(self, inputs):
+        #  The input name in the model signature is `x, hence the export input name is updated.
+        model_inputs = {}
+        model_inputs["x"] = inputs["pixel_values"]
```

Comment:
This is more in an effort of standardization rather than removing ambiguity right? For example if we are to use the exported models with ORTModel later.

Reply:
Yeah intention was both